types/storybook__addon-knobs/angular.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@storybook/addon-knobs/angular' {
```

Comment:
Avoid `declare module` and just write exports directly at the top-level. (The entire file could be just `export * from "./index";`.

Reply:
done in a65fd408d25c20942f489d136079bef4ba879518